Sources/MapboxMaps/Location/Puck/Puck2D.swift

```diff
@@ -313,3 +346,9 @@ private extension Puck2DConfiguration.Pulsing.Radius {
         }
     }
 }
+
+private extension ClosedRange where Bound == Double {
```

Comment:
can we have this extension as `internal` so we don't have a same function in unit tests?